src/command_modules/azure-cli-dls/azure/cli/command_modules/dls/_help.py

```diff
@@ -105,6 +105,43 @@
     short-summary: Deletes a firewall rule in a Data Lake Store account.
 """
 
+helps['dls account virtual-network'] = """
+    type: group
+    short-summary: (PREVIEW) Manage Data Lake Store account virtual network rules.
+"""
+
+helps['dls account virtual-network create'] = """
+    type: command
+    short-summary: Creates a virtual network rule in a Data Lake Store account.
+    parameters:
+        - name: --subnet-id
```

Comment:
This parameter should be called `--subnet` and accept the name or ID of the subnet in question.  You must also expose `--vnet-name` that is used to resolve the subnet when `--subnet` is provided as a name.
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#supporting-name-or-id-parameters